clang/lib/Lex/InitHeaderSearch.cpp

```diff
@@ -347,6 +347,7 @@ void InitHeaderSearch::AddDefaultIncludePaths(
       } else {
         AddPath("/System/Library/Frameworks", System, true);
         AddPath("/Library/Frameworks", System, true);
+        AddPath("/System/Library/SubFrameworks", System, true);
```

Comment:
Should `/System/Library/SubFrameworks` be before `/Library/Frameworks`?

Reply:
Why do you think?

Reply:
The preference should be still SDK content, before we fall-back to `/Library/Frameworks`. There is also the concern that lookups will miss now in `/Library/Frameworks` first, before they hit in `/System/Library/SubFrameworks`.